eng/targets/RepoToolset/ProjectDefaults.props

```diff
@@ -52,6 +49,7 @@
       list as they will not be the same from machine to machine.
     -->
     <AssemblySearchPaths>
+      {HintPathFromItem};
```

Comment:
We specifically deleted this from Roslyn. It allows developers to check in code that depends on machine state.

Reply:
That was a bad decision on arcades part. It was explicitly opting into unreliable builds. Please file a bug to have BuildBoss enforce the removal of hint paths. We should maintain our build as being sane even if arcade allows for flakiness.

Reply:
https://github.com/dotnet/roslyn/issues/31772